### PR TITLE
Integrate Asset Information API

### PR DESCRIPTION
**Pull request by LBHCallumM**
## Link to JIRA ticket

[Story](https://www.pivotaltracker.com/story/show/180329680)

## Describe this PR

Updated GetPropertyUseCase to get property information from Asset API instead of the old API.

I have also added a null check in PropertiesController to return a 404.

#### _Checklist_

- [x] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [x] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### CODE REVIEW

File: RepairsApi.Tests/E2ETests/AssetApiTests.cs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    [SingleThreaded]
+    public class AssetApiTests : MockWebApplicationFactory
+    {
+        public HttpClient Client => CreateClient();
+        private readonly Fixture _fixture = new Fixture();
+
+        [SetUp]
+        public void SetUp()
+        {
+            MockApiGateway.ForcedCode = null;
```
Comment (Asura5) [inline RepairsApi.Tests/E2ETests/AssetApiTests.cs:29]: Just for my benefit - what does this do?

Comment (LBHCallumM) [reply RepairsApi.Tests/E2ETests/AssetApiTests.cs:29]: These are e2e tests related to the getProperty endpoint. I was sticking to the existing naming conventions, but looking back, it doesnt make much sense. I will find a better name for the tests

Comment (LBHCallumM) [reply RepairsApi.Tests/E2ETests/AssetApiTests.cs]: I renamed it to GetPropertyE2ETests.

File: RepairsApi/serverless.yml [hunk 2]
```diff
@@ -96,7 +98,7 @@ resources:
         Path: /${self:service}/${self:provider.stage}/
         RoleName: ${self:service}-lambdaExecutionRole
         AssumeRolePolicyDocument:
-          Version: '2012-10-17'
+          Version: "2012-10-17"
```
[the remark this answers is not in the document]
Comment (LBHCallumM) [reply RepairsApi/serverless.yml]: I blame VSCode for formatting on save. I have reverted everything back.

**Review by Asura5 — APPROVED** [review]

**Pull request merged**